src/components/Cavatica/AnalyzeButton/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { useCallback, useEffect } from 'react';
```

Comment:
This is extracted code from src/views/DataExploration/components/PageContent/tabs/DataFiles/index.tsx so it could be reused in the File Entity page. No change to the logic has been made.